lib/engine/step/g_18_zoo/simple_draft.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require_relative '../base'
+
+module Engine
+  module Step
+    module G18ZOO
+      class SimpleDraft < Base
+        attr_reader :companies
+
+        ACTIONS_WITH_PASS = %w[bid pass].freeze
+
+        def setup
+          @companies = @game.companies.select { |company| company.phase == 'ISR' }.sort
+          @finished = false
+        end
+
+        def available
+          @companies
+        end
+
+        def may_purchase?(_company)
+          true
+        end
+
+        def auctioning; end
+
+        def bids
+          {}
+        end
+
+        def visible?
+          true
+        end
+
+        def players_visible?
+          true
+        end
+
+        def name
+          'Draft'
+        end
+
+        def description
+          'Draft Powers'
+        end
+
+        def finished?
+          @finished
+        end
+
+        def actions(entity)
+          return [] if finished?
+
+          entity == current_entity ? ACTIONS_WITH_PASS : []
+        end
+
+        def process_bid(action)
+          @log << "CHECK if finished: #{@finished} - #{@round.entity_index} #{@game.players.size}"
```

Comment:
Is this a player log message? Or a debug message?

Reply:
debug, I'm going to remove it